content/blog/2020-08-27-august-20-community-gems.md

```diff
@@ -0,0 +1,195 @@
+---
+title: August '20 Community Gems
+date: 2020-08-27
+description: |
+  A roundup of technical Q&A's from the DVC community. This month, we discuss 
+  using CI/CD to validate models, advanced DVC pipeline scenarios, and how CML adds pictures
+  to your GitHub and GitLab comments.
+descriptionLong: |
+  A roundup of technical Q&A's from the DVC community. This month, we discuss 
+  using CI/CD to validate models, advanced DVC pipeline scenarios, and how CML adds pictures
+  to your GitHub and GitLab comments.
+picture: 2020-08-27/Gems_Aug_20.png
+author: elle_obrien
+commentsUrl: https://discuss.dvc.org/t/august-20-community-gems/477
+tags:
+  - Discord
+  - Gems
+  - CML
+  - Hyperparameters
+  - Git LFS
+  - Pipeline
+---
+
+Here are some of our top Q&A's from around the community. With the launch of
+[CML](https://cml.dev) earlier in the month, we've got some new ground to cover!
+
+## DVC questions
+
+### [Q: What's the relationship between your DVC remote and cache? If I have an external cache, do I really need a DVC remote?](https://discordapp.com/channels/485586884165107732/563406153334128681/747588572479094866)
+
+You can think of your DVC remote similar to your Git remote, but for data and
+model artifacts- it's a place to backup and share artifacts, like datasets and
+models, and it gives you methods to push and pull those artifacts to and from
+your team.
+
+Your DVC cache (by default, it's located in `.dvc/cache`) serves a similar
+purpose to a `.git` cache. They're both _local_ caches that optimize access to
```

Comment:
I think we should check and find the proper name/path  for "`.git` cache" - I'm not sure it is the right phrase

Reply:
they both also store these files/data artifacts, different versions/parts of them

Reply:
I've changed: 
```Your DVC cache (by default, it's located in `.dvc/cache`) serves a similar
purpose to your Git objects database (which is by default located in `.git/objects`). They're both _local_ caches that store files (including various versions of them) in a content-addressable format, which helps you quickly checkout different versions to your local workspace. The difference is that `.dvc/cache` is for data/model artifacts, and `.git/objects` is for code. ```

I noted that the Git docs don't call their version of `.dvc/cache` a cache; they call it an objects database https://git-scm.com/book/en/v2/Git-Internals-Git-Objects